packages/devtools/src/ganache/index.ts

```diff
@@ -8,6 +8,7 @@ import {Account} from '../types';
 import {GanacheDeployer} from './deployer';
 import {GanacheNCacheDeployer} from './deployer-with-cache';
 import {GanacheServer} from './server';
+import writeJsonFile = require('write-json-file');
```

Comment:
Is import assignment necessary here?

Reply:
No - I think vscode did that automatically and I didn't notice.